### PR TITLE
Add description for optional extras, or optional equipment of vehicle

### PR DESCRIPTION
**Pull request by adobekan**
Adding attribute for vehicle optional equipment

Signed-off-by: Adnan Bekan <[adnan.bekan@bmwgroup.com](mailto:adnan.bekan@bmwgroup.com)>

### CODE REVIEW

File: spec/Vehicle/Vehicle.vspec [hunk 1]
```diff
@@ -150,6 +150,11 @@ VehicleIdentification.KnownVehicleDamages:
   type: attribute
   description: A textual description of known damages, both repaired and unrepaired.
 
+VehicleIdentification.OptionalExtras:
+  datatype:string[]
```
Comment (erikbosch) [inline spec/Vehicle/Vehicle.vspec:154]: I think the syntax require a blank after the colon, likely reason why build fails

File: spec/Vehicle/Vehicle.vspec [hunk 2]
```diff
@@ -150,6 +150,11 @@ VehicleIdentification.KnownVehicleDamages:
...
+  type: attribute
+  description: Optional extras refers to all car equipment options that are not installed as standard by the manufacturer.
```
Comment (erikbosch) [inline spec/Vehicle/Vehicle.vspec:156]: Do we need to add a comment that allowed values are not standardized, it is up to OEM to define what to use.

Comment (adobekan) [reply spec/Vehicle/Vehicle.vspec:156]: Done.

**Review by SebastianSchildt — APPROVED** [review]
Fine for me, just a note, should this be called "Options" instead, because while probably the intnetion is to add "ZOMGBoomSoundSystem", "Optigrill Seatheating" ) , I would assume there are some "options/extras" that are mandatory/standard in some markets and optional/paid in others (i.e. maybe lane assist).
VSS side, as this is a configuration/technical info that may be helpful, I would assume the intention is to list all options available in a specific vehicle type and only the ones a specific customers in a specific market happened to need to pay extra for??

**Comment by adobekan** [comment]
@SebastianSchildt  Proposal is OptionalExtras and description is saying "options that are not installed as standard ". What is standard I do not see described in this field. DefaultExtras and OptionalExtras you could have. Calling it Options then it will mix everything, plus name I would discuss. :)

**Comment by erikbosch** [comment]
Meeting notes: Please review/comment. Possible merge decision Feb 21st.

**Comment by erikbosch** [comment]
Meeing decision: merge and include in VSS 3.1

**Pull request merged**